src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -0,0 +1,103 @@
+@import "../../patternfly-imports";
+
+.pf-c-app-launcher {
+  // Toggle
+  --pf-c-app-launcher__toggle--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-app-launcher__toggle--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__toggle--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-app-launcher__toggle--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__toggle--MinWidth: var(--pf-global--target-size--MinWidth);
+  --pf-c-app-launcher__toggle--BackgroundColor: transparent;
+  --pf-c-app-launcher__toggle--Color: var(--pf-global--Color--200);
+  --pf-c-app-launcher__toggle--hover--Color: var(--pf-global--Color--100);
+  --pf-c-app-launcher__toggle--active--Color: var(--pf-global--Color--100);
+
+  // Menu
+  --pf-c-app-launcher__menu--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-app-launcher__menu--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-app-launcher__menu--BoxShadow: var(--pf-global--BoxShadow--md);
+  --pf-c-app-launcher__menu--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__menu--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__menu--Top: calc(100% + var(--pf-global--spacer--xs));
+  --pf-c-app-launcher__menu--ZIndex: var(--pf-global--ZIndex--xs);
+
+  // Menu Item
+  --pf-c-app-launcher__menu-item--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__menu-item--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-app-launcher__menu-item--PaddingBottom: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__menu-item--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-app-launcher__menu-item--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-app-launcher__menu-item--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-app-launcher__menu-item--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-app-launcher__menu-item--Color: var(--pf-global--Color--dark-100);
```

Comment:
Why do we set font-size, font-weight, line-height, and color to the defaults?

Reply:
Another inherited style from dropdown. Thought the same thing, want to run it past Sarah first before removing.